Linkta-org/core
# Hotfix: SnackBarNotification Errors

#### Description:
This PR addresses memory issues in `NotificationStore` and includes updates to the `SideNavDrawer` component for better user profile rendering. It also implements a temporary hotfix to remove error notifications in `SideNavDrawer` and `SignInPage`.

#### Key Changes:
1. **NotificationStore Fixes:**
   - Fixed memory leaks by ensuring notification timeouts are properly cleared.
   - Throttled `addNotification` to prevent rapid state changes and reduce memory load.
   - Added a check to prevent exceeding the `MAX_NOTIFICATIONS` limit.

2. **SideNavDrawer Updates:**
   - Improved how the user profile is rendered in the `SideNav`.

3. Temporarily removed error notifications from `SideNavDrawer` and `SignInPage` as a hotfix until a permanent solution is found.

## Type of change




## How Can this be tested? Testing Plan to review this PR



`SideNavDrawer` component
1. Spin up app in firefox
2. Observe browser console and you will not see infinite re-rendering of sideNavDrawer

## Checklist:
